src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="block-img">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
```

Comment:
You can use just images and style only this tag
```suggestion
        <img
          src="../src/images/imac.jpeg"
          alt="imac"
           class="product-card-img
        />
```